browse/static/js/connectedpapers.js

```diff
@@ -0,0 +1,116 @@
+(function () {
+  var $output = $('#connectedpapers-output');
+
+  if ($output.html() != '') {
+    // Toggled off
+    $output.html('');
+    return;
+  }
+
+  $output.html('<p>Loading...</p>');
+
+  const REST_ADDR = 'https://rest.migration.connectedpapers.com/';
+  const CONNECTED_PAPERS_ADDR = 'https://www.connectedpapers.com/';
+  const ARXIV_THUMBNAILS_ADDR = CONNECTED_PAPERS_ADDR + 'arxiv_thumbnails/';
+  const NUMBER_OF_THUMBNAILS = 18;
+  
+  var arxivId = window.location.pathname.split('/').reverse()[0];
+  var arxivIdToCPIdUrl = REST_ADDR + '?arxiv=' + arxivId;
+  var communicationErrorHtml = '<p>Oops, seems like communication with the Connected Papers server is down.</p>';
+  var idNotRecognizedHtml = '<p>Seems like this paper is still not in our database. Please try again in a few days.</p>';
+  
+  
+  $.get(arxivIdToCPIdUrl).done(translationResponse => {
+    if ($output.html() == '') {
+      // Toggled off
+      return;
+    }
+    if (translationResponse == null) {
+      $output.html(idNotRecognizedHtml);
+      return;
+    }
+    var paperId = translationResponse.paperId;
+    var title = translationResponse.title;
+
+    if (paperId.length == 0 || title.length == 0) {
+      $output.html(idNotRecognizedHtml);
+      return;
+    }
+
+    var versionsFetchUrl = REST_ADDR + '?versions=' + paperId;
+
+    $.get(versionsFetchUrl).done(versionsResponse => {
+      if ($output.html() == '') {
+        // Toggled off
+        return;
+      }
+
+      var graphUrl = CONNECTED_PAPERS_ADDR + 'main/' + paperId + '/arxiv';
+      var buildGraphLinkHtml = '<a href="' + graphUrl + '" target="_blank"><p style="margin:0;">View graph for ' + 
```

Comment:
I'd like to suggest a little additional "found" language here. Something like "Connection found. View graph for..."

We are trying to think ahead to when an arxiv user might have multiple services toggles on in each tab. We think leading with a clear "found" message will help users quickly identify relevant info.

Reply:
Thanks for the comment Shamsi!

When a user clicks a paper sometimes the graph will be built for the first time, just for them (we can't build graphs for all existing papers in advance). In rare cases there could be a problem and there will be no graph. Therefore we wouldn't want to promise users that papers were "found" before we actually found them. 

That said we agree that the microcopy here could be more inviting and we suggest the following change:
![image](https://user-images.githubusercontent.com/16747577/105179108-8e11fa00-5b31-11eb-8462-552c064f05e8.png)
(in contrast to what we had now:)

![image](https://user-images.githubusercontent.com/16747577/105179124-979b6200-5b31-11eb-918c-21d1ad544c21.png)

How does that look to you?
Thanks!

Reply:
Hi, I've updated the PR with the design Eddie posted above. Please let us know if you have additional feedback.